tabcmd/locales/de/LC_MESSAGES/de.po

```diff
@@ -0,0 +1,70 @@
+# Tabcmd output strings
```

Comment:
I don't think we need to copy this file across with the english strings - it should fall back on english for strings that aren't found for any given language.